src/main/test/com/inversoft/json/LocaleDeserializerTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2021, Inversoft Inc., All Rights Reserved
+ */
+package com.inversoft.json;
```

Comment:
This project is Apachev2. You can copy the license from another file and update the year.

Reply:
Fixed